### ISSUE TITLE
Snapchat login failure

### ISSUE THREAD

**Issue by sambudda** [Athosone/snapme#2]
Snapchat must have changed some key values as I keep getting login failures for ID and password that works on their latest client. Can you update this project with the latest secret key values or please let me know here? Thank you

![ios simulator screen shot jun 21 2015 10 38 42 am](https://cloud.githubusercontent.com/assets/3536867/8272033/d7e11068-1801-11e5-8ff8-6ada1c52a99a.png)

**Comment by Athosone** [comment]
Hi :)

This is possible that snapchat just secured their api to prevent such app like this.

Anyway it was a project I made to learn iOS development, I will try to look into it later in the week if you want

How did you find my project ?

Cordialement/Best regards,

Werck Ayrton
Student Master II/Etudiant Master II
France, Lyon
Mail: werckayrton@gmail.com
Tel : +33 6 80 25 62 09
Skype : ayrtonwerck

**Comment by sambudda** [comment]
Yes, please do look at it and see if it can be fixed. I am sure its one of the key changes snapchat introduced with v9.10 and it won't be the last. I found your project by pure dumb lucking combing through tons of other snapchat projects in github. Most of of the other iOS were horrible and not maintained for long time. Yours looked somewhat newer and I wanted to try to see if it will work at all or not. 

Never the less please do let me know when you can get to it.

**Comment by sambudda** [comment]
As is the login fails everytime but if I change this value

#define USER_AGENT @"Snapchat/9.10.0.0 (HTC One; Android 4.4.2#302626.7#19; gzip)"

then the code that gets executed is -(void) didPostRequestGetDataSucessed:(NSMutableData *)responseData indexOfSnap:(NSInteger)indexOfSnap

but that whole code is commented out and I am not sure why you have it there?

**Comment by Athosone** [comment]
It was experimental so we tried different way to do a request

Now with experience I would have used AFNetworking

Best regards,
Werck Ayrton
